tests/cmd/e2e/main.go

```diff
@@ -51,6 +52,7 @@ func main() {
 		WebhookServiceName: "webhook-service",
 		WebhookSecretName:  "webhook-secret",
 		WebhookConfigName:  "webhook-config",
+		ImagePullPolicy:    v1.PullIfNotPresent,
```

Comment:
The ImagePullPolicy can be omitted. It is set to `IfNotPresent` by default.

Reply:
If I understand correctly, according to the communication in this morning, `IfNotPresent` is required for e2e test so I think it's better to enforce this policy explicitly.

Reply:
ok